storage/database/db_manager.go

```diff
@@ -379,13 +383,11 @@ func partitionedDatabaseDBManager(dbc *DBConfig) (*databaseManager, error) {
 			fallthrough
 		case StateTrieDB:
 			newDBC := getDBEntryConfig(dbc, entryType, dir)
-			if dbc.NumStateTriePartitions > 1 {
-				db, err = newPartitionedDB(newDBC, entryType, dbc.NumStateTriePartitions)
-			} else {
-				db, err = newDatabase(newDBC, entryType)
-			}
+			newDBC.DBType = DynamoDB
+			db, err = newDatabase(newDBC, entryType)
 		default:
 			newDBC := getDBEntryConfig(dbc, entryType, dir)
+			newDBC.DBType = LevelDB
```

Comment:
Could you explain what this line is for? It is unable to use memorydb, badgerdb or other configured DB.

Reply:
I reverted the change.